atramhasis/views/views.py

```diff
@@ -146,9 +147,12 @@ def concept_view(self):
         scheme_id = self.request.matchdict['scheme_id']
         c_id = self.request.matchdict['c_id']
         provider = self.request.skos_registry.get_provider(scheme_id)
-
         if not provider:
             raise ConceptSchemeNotFoundException(scheme_id)
+        if 'atramhasis.force_display_label_language' in provider.metadata:
+            locale = provider.metadata['atramhasis.force_display_label_language']
+        else:
+            locale = self.request.locale_name
```

Comment:
zou hier ook niet naar de `?language` moeten gekeken worden?

Reply:
wel dat weet ik niet zeker :thinking: die parameter is toegevoegd voor de tree view, maar niet zeker of het de bedoeling is om die altijd toe te laten of dat enkel en alleen een manier was om dit te doen werken

Reply:
Jep, ik volg u hierin.